src/js/views/patients/worklist/filters/date-filter.scss

```diff
@@ -0,0 +1,27 @@
+.datepicker__type-filter {
```

Comment:
I don't think this css should be in this PR.  But I do think there needs to be some spacing for the left/right arrows.

Reply:
Thanks for catching this. I stashed some changes and this lingered.